### PR TITLE
pin to python 3.7.x and add keywords

### PR DESCRIPTION
**Pull request by josephaxisa**

### CODE REVIEW

File: python/setup.py [hunk 1]
```diff
@@ -17,10 +17,11 @@
     license="MIT",
     long_description=open("README.rst").read(),
     long_description_content_type="text/x-rst",
-    keywords=["looker_sdk", "Looker API 3.1"],
+    keywords=["Looker", "Looker API", "looker_sdk", "Looker API 3.1"],
```
Comment (jkaster) [inline python/setup.py]:
```suggestion
    keywords=["Looker", "Looker API", "looker_sdk", "Looker SDK", "SDK", "Looker API 3.1"],
```

Comment (jkaster) [reply python/setup.py]: Only accept the additional tags if you think they make sense

File: python/setup.py [hunk 2]
```diff
@@ -17,10 +17,11 @@
...
     long_description=open("README.rst").read(),
     long_description_content_type="text/x-rst",
-    keywords=["looker_sdk", "Looker API 3.1"],
+    keywords=["Looker", "Looker API", "looker_sdk", "Looker SDK", "SDK", "Looker API 3.1"],
     name=NAME,
     package_data={"looker_sdk": ["py.typed", "looker_sdk/looker-sample.ini"]},
     packages=find_packages(),
+    python_requires=">=3.7.0, <3.8",
     url="https://pypi.python.org/pypi/looker_sdk",
     version=VERSION,
```
Comment (joeldodge79) [inline python/setup.py]: let's bump the version and ship a new package

**Comment by josephaxisa** [comment]
Closes #71

**Pull request merged**